src/CW/Controller/ImageController.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * @file
+ *
+ * Image controller.
+ *
+ * @addtogroup cwentity
+ * @{
+ */
+
+namespace CW\Controller;
+
+/**
+ * Class ImageController
+ * @package CW\Controller
+ */
+class ImageController extends FileController {
+
+  /**
+   * Renders an image.
+   *
+   * @return string
+   *   HTML of the image rendered.
+   *
+   * @throws \Exception
+   */
+  public function render() {
+    return theme('image', [
```

Comment:
Its possible to return render array rather than HTML? Allows the theme layer to be in charge of rendering the item / more future proof for d8